src/app/containers/app-header/app-header.component.ts

```diff
@@ -2,14 +2,13 @@ import { Component, OnDestroy, OnInit } from '@angular/core';
 import { NavigationEnd, Router } from '@angular/router';
 import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { select, Store } from '@ngrx/store';
-import { combineLatest, Observable, of, Subscription } from 'rxjs';
-import { Theme, UserTypeRole } from '../../../app/models/theme.model';
-import { UserDetails } from '../../../app/models/user-details.model';
-
+import { Observable, of, Subscription } from 'rxjs';
+import { first } from 'rxjs/operators';
+import { ApplicationTheme, NavigationItem } from 'src/app/models/theming.model';
```

Comment:
src used here, need to do ../../../app/models/theming.model and same with below

Reply:
Thank you @connorpgpmcelroy, I have now fixed this.